### DISCUSSION TITLE
How to get profile info

### DISCUSSION THREAD

**Discussion by dairoca90** [OpenIdentityPlatform/OpenAM#556] [Q&A]
So i try to get the default claims with the scope profile, the UI is showing me this claims name, given_name, family_name but in the tokeninfo request this values are not coming, am i missing some configuration or something?

"scope":["profile"],"profile":"",

**Comment by maximthomas** [comment]
Hi,
Could you please provide more info - screenshots, http request and response dumps?

**Comment by dairoca90** [comment]
This is a pic from the ui when the user allow the application you can see 3 claims when i request the profile scope

<img width="351" alt="Captura de pantalla 2022-11-15 a la(s) 21 58 49" src="https://user-images.githubusercontent.com/38822740/202080473-be8a6a7e-b606-4f2e-88c5-99639275f7da.png">

But in the tokeinfo response i cannot see those values here is my request and response
```
curl \
--request GET \
"http://am.openam.com:18080/openam/oauth2/realms/operators/tokeninfo?access_token=0b32a757-bc75-4f5e-b9ce-d34e2da3a2c0"


{
    "access_token":"0b32a757-bc75-4f5e-b9ce-d34e2da3a2c0",
    "grant_type":"authorization_code",
    "scope":["profile"],
    "profile":"",
    "realm":"/operators",
    "token_type":"Bearer",
    "expires_in":3598,
    "client_id":"operators"
}

```

Dont know if i have to do something to the OIDC claims script

**Reply by maximthomas** [reply]
@dairoca90 
try to use `userinfo` endpoint
```bash
curl --location --request GET 'http://am.openam.com:18080/openam/oauth2/userinfo' \
--header 'Authorization: Bearer <access_token>'
```
